vyper/codegen/function_definitions/common.py

```diff
@@ -51,7 +51,7 @@ def ir_identifier(self) -> str:
 
     def set_frame_info(self, frame_info: FrameInfo) -> None:
         # XXX: when can this happen?
-        if self.frame_info is not None:
+        if self.frame_info is not None:  # pragma: nocover
```

Comment:
Is this intended to be a compiler panic?

Reply:
no -- i'm not sure why i added the pragma here